DEV-INTRO.md

```diff
@@ -1,59 +1,199 @@
 
-#### List of modules 
+By default docker-compose will use the files `docker-compose.yml` and `docker-compose.override.yml`. We will use the default `docker-compose.yml` file from PEcAn. The `docker-compose.override.yml` file can be used to configure it for your specific environment, in our case we will use it to setup the docker environment for development. Copy the `docker-compose.dev.yml` file to `docker-compose.override.yml` to start working with your own override file, i.e. `cp docker-compose.dev.yml docker-compose.override.yml`. You can now use the command `docker-compose` to launch all the containers setup for development. **The rest of this document assumes you have done this step.**
 
-Installing PEcAn
-----------------
+If you want to start from scratch and remove all old data, but keep your pecan checked out folder, you can remove the folders where you have written the data (see `folders` below). You will also need to remove any of the docker managed volumes. To see all volumes you can do `docker volume ls -q -f name=pecan`. If you are sure, you can either remove them one by one, or use `docker volume rm $(docker volume ls -q -f name=pecan)` to remove them all. **THIS DESTROYS ALL DATA IN DOCKER MANAGED VOLUMES.**. If you changed the path in `docker-compose.override.yml` you will need to remove that volume as well. This will however not destroy the data since it is already on your local machine.
 
-### Virtual Machine
+### First time setup
 
-* Fastest way to get started
-* see PEcAn demo ...
+The steps in this section only need to be done the fist time you start working with the stack in docker. After this is done you can skip these steps. You can find more detail about the docker commands in the [pecan documentation](https://pecanproject.github.io/pecan-documentation/master/docker-index.html).
 
-### Installing from source
+Before doing anything it is recommended to make sure you have the lastest docker images ready. You can do a `docker-compose pull` to get the latest images.
 
-#### From GitHub
+* setup .env file
+* create folders to hold the data
+* load the postgresql database
+* load some test data
+* copy all R packages (optional but recommended) 
+* setup for web folder development (optional)
+
+#### .env file
+
+You can copy the [`env.example`](docker/env.example) file as .env in your pecan folder. The variables we want to modify are:
+
+* `COMPOSE_PROJECT_NAME` set this to pecan, the prefix for all containers
+* `PECAN_VERSION` set this to develop, the docker image we start with
+Both of these variables should also be uncommented by removing the # preceding them. 
+At the end you should see the following if you run the following command `egrep -v '^(#|$)' .env`
+
+```
+COMPOSE_PROJECT_NAME=pecan
+PECAN_VERSION=develop
+```
+
+#### folders
```

Comment:
@robkooper  I'm going through these instructions from scratch to set up this Docker stack on a new server. What I'm actually supposed to _do_ here is now really confusing. From what I can tell, I'm supposed to: 
1. Run `mkdir -p $HOME/volumes/pecan/{lib,pecan,portainer,postgres,rabbitmq,traefik}`
2. Uncomment all of the lines after the `volumes:` line

Is this correct? Do I need to do anything else? 

I think this could be more clear by taking out all of the explanation about why it should be done this way. That content is useful but doesn't really belong here. It should mostly just tell me what to do, and I think the list of volumes with their purpose is also useful and should stay.